backend/prisma/seed.js

```diff
@@ -356,6 +356,493 @@ const medias_metas = [
 ];
 
 
+const levels = [
+	{
+		id: 1,
+		name: "espana",
```

Comment:
Would be possible to have the real names? España instead of espana, as it will be shown in the frontend as such

Reply:
ok